### ISSUE TITLE
DDL tab caches despite refreshes

### ISSUE THREAD

**Issue by olivierlacan** [jakob/Postico#798]
## What did you do?

Views DDL tab on a production table that had received a migration that modified a constraint *after* I first opened Postico and loaded up the table DDL. 

## What did you expect to happen?

When I hit refresh on the DDL tab, I expected the latest table information to be loaded and to see the new constraint pointing to a different table.

## What actually happened?

The DDL kept showing the old table within the constraint statement as `REFERENCES old_table(column)`.

It's not until I switched to the `Structure` tab that the cache was apparently busted and the `new_table` was listed in the constraint.

I think the caching is a bit too aggressive, if I hit `Command` + `R` to `Reload` I expect to bust any cache the view may have. The existence of `Reload Current View Only` sort of tells me that this should have worked by reloading the entire table so maybe it's not a caching issue but an actual bug. I'm not sure.

## What software versions are you using?

Postico version:  1.5.20 (8940)
(select "About Postico" from the Postico menu)

macOS version:  12.0.1 (21A559)
(selecting "About this Mac" from the Apple menu)

PostgreSQL version:  PostgreSQL 12.4 on x86_64-pc-linux-gnu, compiled by x86_64-pc-linux-gnu-gcc (GCC) 7.4.0, 64-bit
(execute the command "SELECT VERSION();" to find out)

**Comment by martinjankoehler** [comment]
@olivierlacan Thanks for reporting this DDL refresh issue!

We've a preview build available at
https://eggerapps-downloads.s3.eu-west-1.amazonaws.com/postico-9203.dmg
Please let us know if this fixes the problem also on your side.

Best regards,
Martin

**Comment by martinjankoehler** [comment]
Closing this issue for now.
Please feel free to re-open or open a new issue in case you discover further problems.
